test/IDE/complete_operators.swift

```diff
@@ -259,9 +260,20 @@ func testInfix15<T: P where T.T == S2>() {
 func testInfix16<T: P where T.T == S2>() {
   T.foo#^INFIX_16^#
 }
+
 func testInfix17(x: Void) {
   x#^INFIX_17^#
 }
+
+// VOID_OPERATORS: Begin completions
```

Comment:
Generally want some feedback as to where the `CHECK:` lines need to go.  Even before this PR seems like the `NO_OPERATORS-NOT` was in a strange place, but could be my misunderstanding.